app/views/admin/reports/show.html.haml

```diff
@@ -5,6 +5,7 @@
   %fieldset.no-border-bottom.print-hidden
     %legend{ align: 'center'}= t(:report_filters)
     = render partial: "admin/reports/filters/#{@report_type}", locals: { f: f }
+    = render partial: "admin/reports/filters/#{@report_subtype}", locals: { f: f } rescue nil
```

Comment:
What are you rescuing here? Can we do a better check in advance?

Reply:
fixed

Reply:
Ah, now I understand. Thank you.